packages/widget-chat/src/containers/chat-widget/index.js

```diff
@@ -83,24 +84,6 @@ export class ChatWidget extends Component {
     }
   }
 
-  /* eslint-disable complexity */
-  /* eslint-disable-reason: Lots of checks for better efficiency, will be broken up later */
-  shouldComponentUpdate(nextProps) {
-    const props = this.props;
-
-    /* eslint-disable operator-linebreak */
-    /* eslint-disable-reason: Giant list of comparisons very difficult to read and diff */
-    return nextProps.conversation.activities !== props.conversation.activities
-      || nextProps.conversation.isLoadingHistoryUp !== props.conversation.isLoadingHistoryUp
-      || nextProps.flags !== props.flags
-      || nextProps.indicators !== props.indicators
-      || nextProps.share !== props.share
-      || nextProps.sparkState.connected !== props.sparkState.connected
```

Comment:
I haven't dove into the redux implementation, but all of the docs seem to indicate that `shouldComponentUpdate` tests should be a last resort. I'll close this as experimental and we can come back to it.